package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ The rules for this file:
  * 2.8.0
 
 Fixes
+ * Fix bug in PCA -- wasn't possible to use `frames=...` syntax before.
```

Comment:
```suggestion
 * Fix bug in PCA preventing use of `frames=...` syntax (PR #4423)
```
We've not been amazing on PR numbers but this is actually super useful when doing the release notes! (mostly so I don't have to edit things afterwards)